blockfinder

```diff
@@ -242,9 +242,13 @@ class DatabaseCache(object):
                'WHERE num_type = ? AND country_code = ? '
                'ORDER BY start_hex')
```

Comment:
Perhaps you could change the select statement to use `SELECT DISTINCT` to avoid having to do duplicate detection in python?